core/src/epicli/data/common/ansible/playbooks/roles/repository/tasks/setup.yml

```diff
@@ -62,12 +62,30 @@
     - not custom_repository_url
     - inventory_hostname in target_repository_hostnames
 
+- name: Copy repository server charts
+  copy:
+    src: "server/charts/"
+    dest: "/tmp/epi-repository-charts"
+    mode: a+x
+  when:
```

Comment:
There is the same condition for multiple tasks, please put them into Ansible block with single condition.